js/background.js

```diff
@@ -77,7 +77,8 @@
     theme: 'sunburst',
     font: 'Inconsolata',
     fontSize: 'medium',
-    lineNumbers: true
+    lineNumbers: true,
+    languageBlacklist: ''
```

Comment:
The problem is that the language is often determined by the content-type. In those cases, url filtering isn't enough.